src/content/overview.md

```diff
@@ -242,7 +242,7 @@ Alternatively, Dart code can be compiled to WebAssembly.
 Dart web contains three compilation modes:
 
 * An incremental JavaScript development compiler enabling a fast developer 
-  cycle.
+  cycle with incremental recompilation (enabling hot reload).
```

Comment:
The text within the sentence is a bit redundant.  Perhaps just "An incremental JavaScript development compiler enabling a fast iteration cycle with stateful hot reload."?